deploy/gcp/main.tf

```diff
@@ -99,9 +99,9 @@ resource "google_container_cluster" "cluster" {
     use_ip_aliases = true
   }
 
+  // See https://kubernetes.io/docs/setup/best-practices/cluster-large/#size-of-master-and-master-components for why initial_node_count is 5. The master node should accommodate up to 100 nodes like this
   remove_default_node_pool = true
-  // see https://github.com/terraform-providers/terraform-provider-google/issues/3385 for why initial_node_count is sum of node counts
-  initial_node_count       = var.pd_count + var.tikv_count + var.tidb_count + var.monitor_count
+  initial_node_count       = 5
```

Comment:
Will the master node get resized when setting different node count for the initial cluster? Previously, we observed that when other node pools created, the cluster is in a blocking state, the master node is upgraded during creation time.